docs/4.0/quickstart.md

```diff
@@ -163,6 +163,31 @@ new-node      xxxxx-xxxx-xxxx-xxxxxxx     10.0.10.2:3022
 !!! tip "NOTE": 
     Teleport also supports static pre-defined invitation tokens which can be set in the [configuration file](admin-guide.md#adding-nodes-to-the-cluster)
 
+## Adding a node located behind NAT
+With the current setup you've only been able to add nodes that have direct acess to the 
+authentication server and within the internal IP range of the cluster. We recommend 
+setting up a [Trusted Cluster](../admin-guide/#trusted-clusters) if you have workloads split
+ across different networks / clouds. 
+
+Teleport Node Tunneling lets you add a node to an exsiting Teleport Cluster. This can be 
+useful for IoT applications or for managing a couple of servers in a different network.  
+
+Similar to [Adding Nodes to Cluster](/quickstart/#adding-nodes-to-cluster), use `tctl` to
+create a single-use token fro a node, but this time you'll replace the auth-server IP with
+the URL of the Proxy Server. In the Example below, we've replaced the Auth IP to the Proxy
+server address `teleport.example.com`.  
+
+```bash
+$ sudo tctl nodes add
+
+The invite token: n92bb958ce97f761da978d08c35c54a5c
+Run this on the new node to join the cluster:
+teleport start --roles=node --token=n92bb958ce97f761da978d08c35c54a5c --auth-server=teleport-proxy.example.com
+```
+
+!!! tip "NOTE": 
+    The Proxy will require port `3025` to be open so it can add the new node to the Auth server. 
```

Comment:
@russjones Is this true? I figured the only connection that would be necessary is for the node to be able to connect back to the proxy - the proxy would handle connecting to the auth server itself?

Reply:
I found this out when testing on GCP, my firewall was pretty locked down. I noticed it kept trying to connect to port 3025 ( SSH port used by the Auth Service to serve its API to other nodes in a cluster. ) when it first made it's connection.

Reply:
Using the ports in the default configuration, the node needs to be able to talk to ports 3080 and 3024 on the proxy. 

Port 3080 is used to initially fetch the credentials (SSH and TLS certificates) and for discovery (where is the reverse tunnel running, in this case 3024).

Port 3024 is used to establish a connection to the Auth Server through the proxy.

Reply:
I ended out removing the note, and left some more information about how the ports are used.